nextflow_schema.json

```diff
@@ -365,36 +389,44 @@
                         "loess",
                         "none"
                     ],
-                    "description": "DIAlignR XICfilter parameter"
+                    "description": "DIAlignR XICfilter parameter",
+                    "fa_icon": "fas fa-funnel-dollar"
                 },
                 "DIAlignR_parallelization": {
                     "type": "boolean",
                     "default": true,
-                    "description": "Whether DIAlignR should be executed using multithreading (may cause errors)"
+                    "description": "Whether DIAlignR should be executed using multithreading (may cause errors)",
```

Comment:
is this still the case for the current version?

Reply:
Not sure, but maybe its good to have this option, in case there problems